src/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body,
+a,
+div,
+li,
+p,
+h2,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
Please, pay attention to the comments, you need to create different css files for your blocks, reset and normalize